src/kaggle/models/kaggle_models_extended.py

```diff
@@ -115,9 +115,14 @@ def __repr__(self):
 class File(object):
 
   def __init__(self, init_dict):
-    parsed_dict = {k: parse(v) for k, v in init_dict.items()}
-    self.__dict__.update(parsed_dict)
-    self.size = File.get_size(self.totalBytes)
+    try: # TODO Remove try-block

```

Comment:
what's the issue with try catch, add context

Reply:
Again, this is just another note-to-self to address after Swagger is removed.